README.md

```diff
@@ -1518,6 +1523,14 @@ age > 18 ? location.assign("continue.html") : stop = true;
 
 **[⬆ back to top](#table-of-contents)**
 
+## Logging
+
+Always use a wrapper for logging rather than using `console.log();`. This is due to some browsers crashing when `console.log();` is used.
+
+[This tool](https://github.com/patik/console.log-wrapper) is a very good wrapper with some great customizable options.
+
```

Comment:
We actually use https://github.com/pimterry/loglevel in fesk. Maybe something more along the lines of:

"Always use a wrapper for logging rather than using `console.log()`. FESK uses [loglevel](https://github.com/pimterry/loglevel) by default and angular has [$log](https://docs.angularjs.org/api/ng/service/$log) built in."